src/main/java/uk/gov/hmcts/ccd/domain/service/aggregated/MergeDataToSearchResultOperation.java

```diff
@@ -120,7 +120,7 @@ private SearchResultViewItem buildSearchResultViewItem(final CaseDetails caseDet
                                                            final CaseTypeDefinition caseTypeDefinition,
                                                            final SearchResultDefinition searchResult) {
         Map<String, JsonNode> caseData = new HashMap<>(caseDetails.getData());
-        Map<String, Object> caseMetadata = new HashMap<>(caseDetails.getMetadata());
+        Map<String, Object> caseMetadata = new HashMap<>(caseDetails.getFormattedMetadata());
```

Comment:
Change not reflected in unit test.

Reply:
There is not test in caseDetails  but I added a test  to evaluate the case reference type in MergeDataToSearchResultOperation.java